src/SafetyMonitor.cpp

```diff
@@ -0,0 +1,127 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Safety Node
+//
+// This node monitors other nodes for failures using bonds
+//
+// Note that a bond has to be created in both the watched node, and this
+// safety node with matching names and IDs (See the Bond Example region below) 
+//
+// Bond should already exist as a package in ROS. Docs at:
+// http://docs.ros.org/api/bondcpp/html/classbond_1_1Bond.html
+////////////////////////////////////////////////////////////////////////////
+
+#include <ros/ros.h>
+#include <bondcpp/bond.h>
+
+#include "std_msgs/String.h"
+
+// Heartbeat rate for bonds
+const float kHeartbeatSec = 0.2;
+const float kTimeoutSec = 0.5;
+
+// Rate in hz to check the bonds. Make it loop three times faster than the heartbeats
+const float kLooptime = 1.0/(kHeartbeatSec/3.0);
```

Comment:
This (and the two variables above it) need some clarification as to whether they're in Hz or seconds.  If kHeartbeatSec is in seconds, then the comment above it should change and this variable should be kLoopFrequency